### PR TITLE
Update network settings to node 2.10 and explorer backend 1.17

### PR DESCRIPTION
**Pull request by nop33**
TODO:
- [x] Make sure node endpoints are available (I already asked Benoit):
   - https://node-v210.mainnet.alephium.org/infos/version
   - https://node-v210.testnet.alephium.org/infos/version

This PR:
1. Updates the network settings in desktop and mobile wallet to use the latest version-pinned endpoints node v2.10 and explorer backend 1.1.7
2. Adds tests for desktop wallet network settings migration
3. Sets up tests for mobile wallet

I tried to write migration tests for the mobile wallet but I didn't manage to find a way to mock the `expo-secure-store`. I managed to set up tests for the mobile wallet following the [expo guide](https://docs.expo.dev/develop/unit-testing/) and the [jest-expo guide](https://www.npmjs.com/package/jest-expo). I did not find any info on mocking expo-secure-store in [its documentation page](https://docs.expo.dev/versions/latest/sdk/securestore/). StackOverflow [didn't](https://stackoverflow.com/questions/73369848/expo-secure-store-s-securestore-setitemasync-and-getitemasync-not-working-with-j) [help](https://stackoverflow.com/questions/66341071/expo-secure-store-unavailable-in-jest-tests/66341640#66341640) nor this [GH issue](https://github.com/expo/expo/issues/4298). Then I looked into [mocking native modules](https://docs.expo.dev/modules/mocking/) and I installed `sourcekitten` in my system to try and generate mocks for expo-secure-store, but when I'd run the command `npx expo-modules-test-core generate-ts-mocks` from the expo-secure-store path, I'd get an error `Error: Cannot find module 'typescript'`. I also noticed that other expo packages already come with a `mocks` folder (like the [expo-clipboard](https://github.com/expo/expo/tree/main/packages/expo-clipboard)), but [expo-secure-store doesn't have such a folder](https://github.com/expo/expo/tree/main/packages/expo-secure-store). Lastly, I tried to manually mock the functions of expo-secure-store with jest, but after following [the docs to mock modules](https://jestjs.io/docs/mock-functions#mocking-modules) I couldn't make it work (`mockResolvedValue` would not be available).

### CODE REVIEW

**Comment by nop33** [comment]
@mvaivre the endpoints are now ready! Ready for review

File: apps/mobile-wallet/__tests__/test.test.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+it('works', () => {
```
Comment (mvaivre) [inline apps/mobile-wallet/__tests__/test.test.ts:19]: Out of curiosity: why is this needed? If tests don't work we'll see them failing somehow, no?

**Review by mvaivre — APPROVED** [review]
LGTM! 👍  Just one tiny question.

File: apps/mobile-wallet/__tests__/test.test.ts (hunk 1 above)
Comment (nop33) [reply apps/mobile-wallet/__tests__/test.test.ts:19]: haha, yes, of course! But look at the test, do you want this to fail? :P 

```
expect(1).toBe(1)
```

I am simply setting up the test suite here, as I described in the PR I could not find a way to test expo secure store

**Pull request merged**